docker-images/kafka/scripts/kafka_config_generator.sh

```diff
@@ -82,6 +102,9 @@ listener.name.replication.ssl.keystore.location=/tmp/kafka/cluster.keystore.p12
 listener.name.replication.ssl.truststore.location=/tmp/kafka/cluster.truststore.p12
 listener.name.replication.ssl.client.auth=required
 
+sasl.enabled.mechanisms=$(echo -e "$SASL_ENABLED_MECHANISMS" | uniq | awk -vORS=, '/.+/{ print $1 }' | sed 's/,$/\n/')
```

Comment:
just to be sure, does the `uniq` command delete the "SCRAM-SHA-512" value that we can have twice if scram-sha-512 is enabled on both client and clienttls listeners, right?